pkg/nuctl/command/get.go

```diff
@@ -232,7 +232,7 @@ func newGetAPIGatewayCommandeer(getCommandeer *getCommandeer) *getAPIGatewayComm
 
 			commandeer.getAPIGatewaysOptions.Namespace = getCommandeer.rootCommandeer.namespace
 
-			apiGateways, err := getCommandeer.rootCommandeer.platform.GetAPIGateways(&commandeer.getAPIGatewaysOptions)
+			apiGateways, err := getCommandeer.rootCommandeer.platform.GetAPIGateways(context.Background(), &commandeer.getAPIGatewaysOptions)
```

Comment:
for all nuctl commands + ctx - need to get the `context` from the root commandeer